view/adminhtml/templates/info/info.phtml

```diff
@@ -16,17 +17,27 @@ $title = $payment->getMethodInstance()->getTitle();
 
 <h4>
     <span class="acquired-title">Acquired Payments - <?= $title; ?></span>
-    <span class="acquired-card-icon"><img width="50" height="30" src="<?= $this->getCardIcon((string)$payment->getCcType()); ?>" title="<?= $title; ?>" alt="<?= $payment->getCcType(); ?>"/></span>
+    <?php if($payment->getMethod() == 'acquired_card') : ?>

```

Comment:
Should there be a set of payment method constants somewhere that we can reference? Or a `$payment->isCardPayment()` style method?

- [ ] Improvement to payment method identification

Reply:
i can add a constant check here, we cannot use isCardPayment() as the $payment is a generic class from Magento 2 rather than a typed instance of our type. The exact instance is usually accessed via a different factory class, so I would suggest going with the constant instead.